test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/SystemdActivation/docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Ensure that dotnet is added to the PATH
+scriptDir=$(dirname "${BASH_SOURCE[0]}")
+repoDir=$(cd $scriptDir/../../.. && pwd)
```

Comment:
You can replace `cd $scriptDir/../../.. && pwd` with `readlink -e $scriptDir/../../..`